src/twisted/protocols/test/test_tls.py

```diff
@@ -1846,161 +1871,3 @@ def acceptableProtocols(self):
         @rtype: L{list} of L{bytes}
         """
         return self._acceptableProtocols
-
-
-class IProtocolNegotiationFactoryTests(TestCase):
```

Comment:
@exarkun  this was moved for a separate ticket https://twistedmatrix.com/trac/ticket/10067